types/tar-js/index.d.ts

```diff
@@ -0,0 +1,25 @@
+// Type definitions for tar-js 0.3
+// Project: http://github.com/beatgammit/tar-js
+// Definitions by: Narazaka <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/tar-js/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me